apis/python/node/src/lib.rs

```diff
@@ -197,21 +233,43 @@ impl Node {
 }
 
 /// Start a runtime for Operators
+///
+/// :rtype: None
 #[pyfunction]
 pub fn start_runtime() -> eyre::Result<()> {
     dora_runtime::main().wrap_err("Dora Runtime raised an error.")
 }
 
+/// Dora Status for dora python operators.
+/// :rtype: DoraStatus
+#[derive(Copy, Clone, Debug, PartialEq)]
+#[pyclass]
+pub enum DoraStatus {
+    CONTINUE,
+    STOP,
+    STOP_ALL,
```

Comment:
We should silence the "variant `STOP_ALL` should have an upper camel case name" warning before merging.

Reply:
Sorry. I was actually planning to remove this and use the previous version as `__init__.py` was mandatory with `__init__.pyi`